Documentation/learning/why.md

```diff
@@ -0,0 +1,6 @@
+# Why etcd
+
+etcd is designed to store a small amount of metadata in consistent, fault-tolerant way. Distributed systems use etcd as the root of scheduling and service discovery configuration storage. [Distributed lock][etcd-concurrency] is implemented on top of etcd, useful for master election in other distributed databases.
```

Comment:
what's meant to be difference between this and https://github.com/coreos/etcd/blob/master/README.md#etcd ?